schema.sql

```diff
@@ -11,4 +11,26 @@ CREATE TABLE animals (
 
 /* Altered animals table and add new column*/
 
-ALTER TABLE animals ADD COLUMN species varchar;
\ No newline at end of file
+ALTER TABLE animals ADD COLUMN species varchar;
+
+/* Create owners table */
+CREATE TABLE owners (
+ id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+ full_name VARCHAR(50),
+ age INT
+);
+
+/* Create species table */
+CREATE TABLE species (
+ id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+ name VARCHAR(50)
+);
+
+/* Modify primary key */
+
```

Comment:
Kindly remove the `species` column as the requirement states 👇 

![](https://i.imgur.com/ImfL7qN.png)

Reply:
Hello @DammyShittu  Your comment doesn't seem to be clear. please shade more light

Reply:
Thanks its been implemented